src/Console/Commands/ForgeDeployCommand.php

```diff
@@ -47,14 +47,14 @@ public function __construct()
     public function handle()
     {
         if (!$this->isForced()) {
-            $goAhead = $this->confirm('Whould you like to trigger deploy on Forge server?');
+            $goAhead = $this->confirm('Would you like to trigger deploy on Forge server?');
 
             if (!$goAhead) {
                 return;
             }
         }
 
-        $deploymentUrl = env('FORGE_DEPLOY_URL');
+        $deploymentUrl = config('FORGE_DEPLOY_URL');
```

Comment:
I swapped out this to use config just in case a consumer is caching their configuration files:

> If you execute the `config:cache` command during your deployment process, you should be sure that you are only calling the env function from within your configuration files. Once the configuration has been cached, the `.env` file will not be loaded and all calls to the env function will return `null`.

https://laravel.com/docs/8.x/helpers#method-env